pkg/obfuscate/obfuscate.go

```diff
@@ -0,0 +1,224 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package obfuscate implements quantizing and obfuscating of tags and resources for
+// a set of spans matching a certain criteria.
+package obfuscate
+
+import (
+	"bytes"
+	"sync/atomic"
+
+	"github.com/DataDog/datadog-go/statsd"
+)
+
+//go:generate easyjson -no_std_marshalers $GOFILE
```

Comment:
Not sure what this does, does it have any implications when we use this code as a library?

Reply:
No. It simply generates the code in obfuscate_easyjson.go once, and that's it. We could even remove this line and it'd still be fine as long as we leave the generated file.

Reply:
Being able to regenerate is good :)